### ISSUE TITLE
NeRF results

### ISSUE THREAD

**Issue by keloee** [wyc-Chang/ImMPI#1]
Thanks for sharing a great work.
Currently, I'm trying to reproduce your results about your training with the original NeRF, but I cannot produce your results. Can you share this NeRF training code.

**Comment by SunshineWYC** [comment]
The contrast method NeRF is implemented with NeRF-pytorch version, which can be found on github. The parameters adjusted during training process include iterations, scene range and camera parameter data loading file. The training device is RTX3090.
Make sure to convert the camera extrinsic correctly.&nbsp;
Good luck.



---Original---
From: ***@***.***&gt;
Date: Mon, Feb 6, 2023 12:49 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [wyc-Chang/ImMPI] NeRF results (Issue #1)




 
Thanks for sharing a great work.
 Currently, I'm trying to reproduce your results about your training with the original NeRF, but I cannot produce your results. Can you share this NeRF training code.

**Comment by keloee** [comment]
I am having trouble converting the camera extrinsic by the following code.
`opencv2blender = np.array([[1, 0, 0, 0], [0, -1, 0, 0], [0, 0, -1, 0], [0, 0, 0, 1]])`
`pose = np.linalg.inv(pose) @ opencv2blender`

Is there something I am missing and If it's possible to provide me with your data-loading file?

Best regards!

**Comment by SunshineWYC** [comment]
This conversion is from opencv coordinate definition to blender coordinate cause the axis definition is different in these two system. Actually the definition of coordinate in Levir-NVS Dataset is same with nerf-pytorch project. It should be noted that the extrinsic in Levir-NVS is w2c, while c2w in nerf-pytorch project, so inversion operation is needed after load the raw extrinsic matrix.



---Original---
From: ***@***.***&gt;
Date: Mon, Feb 6, 2023 15:29 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [wyc-Chang/ImMPI] NeRF results (Issue #1)




 
I am having trouble converting the camera extrinsic by the following code.
 opencv2blender = np.array([[1, 0, 0, 0], [0, -1, 0, 0], [0, 0, -1, 0], [0, 0, 0, 1]]) pose = np.linalg.inv(pose) @ opencv2blender`
 
Is there something I am missing and If it's possible to provide me with your data-loading file?
 
Best regards!
